shardingsphere-test/shardingsphere-integration-test/shardingsphere-integration-test-env/src/test/java/org/apache/shardingsphere/test/integration/env/container/atomic/storage/config/impl/opengauss/OpenGaussContainerConfiguration.java

```diff
@@ -17,27 +17,22 @@
 
 package org.apache.shardingsphere.test.integration.env.container.atomic.storage.config.impl.opengauss;
 
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
 import org.apache.shardingsphere.test.integration.env.container.atomic.storage.config.StorageContainerConfiguration;
-import org.testcontainers.shaded.com.google.common.collect.ImmutableMap;
 
 import java.util.Map;
 
-public class DefaultOpenGaussContainerConfiguration implements StorageContainerConfiguration {
+/**
+ * OpenGauss container configuration.
+ */
+@Getter
+@RequiredArgsConstructor
```

Comment:
Constructor annotation should in the front of field annotation

Reply:
ok, remember